### DISCUSSION TITLE
Loan vs Mortgage

### DISCUSSION THREAD

**Discussion by M1ck0** [firefly-iii/firefly-iii#9403] [Firefly III] [answered]
I took money from the bank to buy a house, and now I pay my bank each month. How should this be added to Firefly? As a loan? Mortgage?

Also, how should I make payments towards this loan/mortgage?

As this is description for loan - `This field is purely cosmetic and won't be calculated for you. As it turns out banks are very sneaky so Firefly III never gets it right.` - is it possible to make this work? Maybe I (user) can give a formula for this based on our bank that Firefly would use? Is this possible? 

Thanks

**Comment by JC5** [comment, marked as answer]
They are the same thing, it's just a title.

https://docs.firefly-iii.org/how-to/firefly-iii/finances/liabilities/

A custom formula will not be an option I'm afraid :)

**Reply by M1ck0** [reply]
Got it. Thanks for the info
